Microsoft.Toolkit.Graph.Controls/Controls/GraphPresenter/GraphPresenter.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Microsoft.Graph;
+using Microsoft.Toolkit.Uwp.Helpers;
+using Newtonsoft.Json.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Graph.Controls
+{
+    /// <summary>
+    /// Specialized <see cref="ContentPresenter"/> to fetch and display data from the Microsoft Graph.
+    /// </summary>
+    public class GraphPresenter : ContentPresenter
+    {
+        /// <summary>
+        /// Gets or sets a <see cref="IBaseRequestBuilder"/> to be used to make a request to the graph. The results will be automatically populated to the <see cref="ContentPresenter.Content"/> property. Use a <see cref="ContentPresenter.ContentTemplate"/> to change the presentation of the data.
+        /// </summary>
+        public IBaseRequestBuilder RequestBuilder
+        {
+            get { return (IBaseRequestBuilder)GetValue(RequestBuilderProperty); }
+            set { SetValue(RequestBuilderProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="RequestBuilder"/> dependency property.
+        /// </summary>
+        /// <returns>
+        /// The identifier for the <see cref="RequestBuilder"/> dependency property.
+        /// </returns>
+        public static readonly DependencyProperty RequestBuilderProperty =
+            DependencyProperty.Register(nameof(RequestBuilder), typeof(IBaseRequestBuilder), typeof(GraphPresenter), new PropertyMetadata(null));
```

Comment:
What if the RequestBuilder changes after the GraphPresenter is loaded? Shouldn't this be handled on the DP's changed event?

Reply:
https://github.com/windows-toolkit/Graph-Controls/issues/56